forte/data/data_store.py

```diff
@@ -939,6 +943,49 @@ def _parse_onto_file(self):
             return
         raise NotImplementedError
 
+    @staticmethod
+    def get_class(type_name):
+        r"""Returns the ontology class based on class name. Only a class that exists in current
+        DataStore will be returned.
+
+        Args:
+            type_name (str): Fully qualified name of the class.
+        Returns:
+            The target class.
+
+        Raises:
+            ValueError: If class is not found based on :attr:`class_name` in DataStore.
+        """
+        if type_name not in DataStore._type_attributes:
+            raise ValueError(f"Class not found in DataStore: {type_name}")
+        return get_class(type_name)
+
+    @staticmethod
+    def get_subtypes(type_name):
+        r"""Returns a complete list of subclasses based on class name.
+
+        Args:
+            type_name (str): Fully qualified name of the class.
+        Returns:
+            A list of subtypes.
+        """
+        if DataStore.enable_cache:
+            try:
+                return DataStore._get_subtype_cache[type_name]
+            except KeyError:
+                pass
+
+        cls = DataStore.get_class(type_name)
+        subtypes = []
+        for subtype in DataStore._type_attributes:
+            if DataStore._is_subclass(subtype, cls):
+                subtypes.append(subtype)
+        subtypes.sort()
+        DataStore.enable_cache = True
```

Comment:
I don't understand this `enable_cache` switch here, what can it do? It is like this switch is always on after you first call this function.